awx/ui/client/src/instance-groups/container-groups/edit-container-group.controller.js

```diff
@@ -33,30 +33,59 @@ function EditContainerGroupController($rootScope, $scope, $state, models, string
   vm.form.credential._displayValue = EditContainerGroupDataset.data.summary_fields.credential.name;
   vm.form.credential.required = true;
   vm.form.credential._value = EditContainerGroupDataset.data.summary_fields.credential.id;
-  vm.podSpec = {
-    type: 'textarea',
-    id: 'pod_spec'
-  };
 
-  vm.podSpec.label = strings.get('container.POD_SPEC_LABEL');
+  vm.tab = {
+    details: {
+            _active: true,
+            _go: 'instanceGroups.editContainerGroup',
+            _params: { instance_group_id: instanceGroup.get('id') }
+    },
+    instances: {
+        _go: 'instanceGroups.containerGroupInstances',
+        _params: { instance_group_id: instanceGroup.get('id') }
+    },
+    jobs: {
+        _go: 'instanceGroups.containerGroupJobs',
+        _params: { instance_group_id: instanceGroup.get('id') }
+    }
+};
 
   vm.form.extraVars = {
     label: strings.get('container.POD_SPEC_LABEL'),
-    value: EditContainerGroupDataset.data.pod_spec_override,
-    name: 'extraVars'
+    value: EditContainerGroupDataset.data.pod_spec_override || instanceGroup.model.OPTIONS.actions.PUT.pod_spec_override.default,
+    name: 'extraVars',
+    toggleLabel: strings.get('container.POD_SPEC_TOGGLE')
   };
 
+  if (EditContainerGroupDataset.data.pod_spec_override) {
```

Comment:
When loading / reloading the edit page want to represent the `customize pod spec` toggle as OFF when the user-provided pod spec is identical to the default one.

Changing these lines to the snippet below will give us the desired behavior:

```js
  const podSpecValue = EditContainerGroupDataset.data.pod_spec_override.trim();
  const defaultPodSpecValue = instanceGroup.model.OPTIONS.actions.PUT.pod_spec_override.default.trim();

  if (podSpecValue && podSpecValue !== defaultPodSpecValue) {
    vm.form.extraVars.isOpen = true;
  } else {
    vm.form.extraVars.isOpen = false;
  }
```